lang/fr.xml

```diff
@@ -453,6 +453,8 @@ Une fois le téléchargement terminé, Vita3K redémarrera automatiquement et in
 		<later_version_already_installed>Une version plus récente de Vita3K est déjà installée.</later_version_already_installed>
 		<latest_version_already_installed>La dernière version de Vita3K est déjà installée.</latest_version_already_installed>
 		<new_features>Nouveautés de la version {}</new_features>
+		<author>Author</author>
```

Comment:
```suggestion
		<authors>Auteurs</authors>
```

Reply:
I have write error in code i jyts see here, is Authors* need change it 😅

Reply:
okay, i got it

Reply:
okay, i got it